lib/css/components/description.less

```diff
@@ -0,0 +1,9 @@
+.s-description {
```

Comment:
Good call pulling this one out. We use this "component" in a few places, but never really document it.